modules/swagger-models/src/main/java/io/swagger/models/refs/GenericRef.java

```diff
@@ -91,12 +93,18 @@ private static String computeSimpleRef(String ref, RefFormat format, RefType typ
 
     private static RefFormat computeRefFormat(String ref) {
         RefFormat result = RefFormat.INTERNAL;
+
         if (ref.startsWith("http:") || ref.startsWith("https:")) {
             result = RefFormat.URL;
         } else if (ref.startsWith("#/")) {
             result = RefFormat.INTERNAL;
         } else if (ref.startsWith(".") || ref.startsWith("/")) {
             result = RefFormat.RELATIVE;
+        }else if (!ref.contains(":") &&   // No scheme
+                !ref.startsWith("#") && // Path is not empty
+                !ref.startsWith("/")&& // Path is not absolute
+                StringUtils.countMatches(ref, ".") == 1) { // Path does not start with dot but contains "." (file extension)
```

Comment:
We try not to add deps if not strictly necessary, use instead indexOf / charAt directly in code, probably `indexOf(".") != -1` is sufficient